MovingBlocks/Terasology
# Correct incorrectly-halved collision extents in basic shapes

### Contains

Fixes #3746 - as well as wrong collision & selection boxes for slabs, eighth blocks, and offset blocks

### How to test

Place or find any block using the stair, half block, eighth block, vertical offset plane, or horizontal offset plane shapes, and ensure that player collisions and cursor selections occur in the proper areas.
